src/app/about/page.tsx

```diff
@@ -16,13 +120,21 @@ export default function About() {
           careers in software development.
         </p>
       </div>
-      <Image
-        className={styles.image}
-        src="fall_2022_planning_meeting.jpg"
-        alt="SSD members (about 30) at our Fall 2022 planning meeting in CISE."
-        height={600}
-        width={600}
-      />
+      <div className={styles.imageContainer}>
+        <Image
+          className={styles.image}
+          src="fall_2022_planning_meeting.jpg"
+          alt="SSD members (about 30) at our Fall 2022 planning meeting in CISE."
+          height={600}
+          width={600}
+        />
+      </div>
+      <section className={styles.officers}>
+        <div className={styles.container}>
```

Comment:
[Comment]: Will probably have to abstract this down the line when we add past officers. Don't change anything here; just making note and musing for the future.